<?xml version="1.0" ?>
<!--
Loads five Prius models and one world model onto the ROS parameter server.
-->
<launch>
  <param name="car_count" type="int" value="5" />

  <param name="car_description_1"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/drake_prius/xacro/drake_prius_free_in_world_1.xacro'" />

  <param name="car_description_2"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/drake_prius/xacro/drake_prius_free_in_world_2.xacro'" />

  <param name="car_description_3"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/drake_prius/xacro/drake_prius_free_in_world_3.xacro'" />

  <param name="car_description_4"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/drake_prius/xacro/drake_prius_free_in_world_4.xacro'" />

  <param name="car_description_5"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/drake_prius/xacro/drake_prius_free_in_world_5.xacro'" />

  <param name="world_description"
         command="$(find xacro)/xacro.py '$(find drake_cars_examples)/models/planar_world.xacro'" />
</launch>